Mage.Client/src/main/java/mage/client/SessionHandler.java

```diff
@@ -1,8 +1,6 @@
 package mage.client;
 
 import mage.cards.decks.DeckCardLists;
-import mage.client.chat.LocalCommands;
```

Comment:
Adding this here so that I can make a thread.

While trying to benchmark this, I was running a local server with a 4 player commander game. Me and 3 AI. I immediately concede to let the AI play while I watch. At some point the game gameview crashes and I'm thrown back to the tables view.
The error is a `ConcurrentModificationException` from inside Netty. I am attaching both the client and the server logs.
[server.txt](https://github.com/magefree/mage/files/9691024/server.txt)
[client.txt](https://github.com/magefree/mage/files/9691025/client.txt)

I am running on windows 10 and my JDK is corretto-1.8.0_342